heron/config/src/yaml/BUILD

```diff
@@ -14,7 +14,7 @@ filegroup(
 
 filegroup(
     name = "conf-yaml",
-    srcs = glob(["conf/**/*.yaml"] + ["conf/**/*.aurora"]),
+    srcs = glob(["conf/**/*.yaml"] + ["conf/**/*.aurora"] + ["conf/**/*.sh"]),
```

Comment:
Hmm. Is it possible to avoid the .sh files here?

Reply:
We need the heron_nomad.sh script to start heron_executors in nomad.  Not sure where else would be more appropriate to put this since the script needs to be available on where you install the heron cli.  The heron.aurora script is placed in the aurora config folder.

Reply:
heron.aurora might be ok (maybe not ideal though) because aurora file is a config and used by aurora cli.

.sh file is worse than aurora file. :(  So I am wondering if it is possible to avoid this .sh file and have a config to be used by nomad CLI?

I don't know nomad enough and I am open to other thoughts/suggestions. Also this is not a blocker if this is not feasible.

Reply:
The .sh file is actually read by the scheduler and submitted to nomad as part of a job description in json format.  I can inline the contents of the .sh script in the scheduler code, but that might not be as clear to the user as just having a file with comments in it

Reply:
kk.